python/tskit/tables.py

```diff
@@ -1717,6 +1741,9 @@ def __init__(self, max_rows_increment=0, ll_table=None):
             ll_table = _tskit.SiteTable(max_rows_increment=max_rows_increment)
         super().__init__(ll_table, SiteTableRow)
 
+    def _has_discrete_coords(self):
+        return np.all(self.position) == np.floor(self.position)
```

Comment:
No coverage here as "position" needs to be in the tests.

Reply:
Oh, of course. Doh! sorry.